boot/duneboot.ml

```diff
@@ -1167,7 +1185,12 @@ module Library = struct
         ~os_type
     =
     let files = scan ~dir ~include_subdirs in
-    let modules = modules files ~build_info_module ~root_module in
+    let modules =
+      modules files ~build_info_module ~root_module
+      |> Module_name.Map.to_list
```

Comment:
This get's introduced in 5.1. You probably want to use the old name `Map.bindings`.